### ISSUE TITLE
pill mentions don't ping users if they're all mashed together

### ISSUE THREAD

**Issue by non-Jedi** [vector-im/element-meta#1617]

**Comment by t3chguy** [comment]
This will be fixed by Intentional Mentions cc @clokep

**Comment by clokep** [comment]
Yes, after [MSC3952](https://github.com/matrix-org/matrix-spec-proposals/pull/3952) mentions won't be parsed from the text body, but from metadata attached to the event. We can close this.
